streamer/test/unit/streamer/test_server.py

```diff
@@ -13,7 +13,7 @@
 MODULE_PREFIX = 'pulp.streamer.server.'
 
 
-class TestStreamerListener(TestCase):
+class TestStreamerListener(unittest.TestCase):
```

Comment:
Oh, there were several asserts I was using (`assertIn`, `assertIs`) which I thought I could use in rhel6 with our compat module. However, that was not the case. I updated them to `assertTrue` in my second test PR.